go.mod

```
@@ -1,41 +1,42 @@
 module github.com/quortex/kubestatic
 
-go 1.22.4
+go 1.23.0
+
+toolchain go1.23.7
```

```suggestion
go 1.23.0
```

Also could you run `go mod tidy` ?